spec/TillSpec.js

```diff
@@ -0,0 +1,93 @@
+describe('Till', function() {
+  var till;
```

Comment:
If you declare the var outside of the function scope, use a before hook or just make a new one per function